appium/webdriver/webdriver.py

```diff
@@ -327,6 +339,43 @@ def find_elements_by_android_uiautomator(self, uia_string):
         """
         return self.find_elements(by=By.ANDROID_UIAUTOMATOR, value=uia_string)
 
+    def find_element_by_image(self, png_img_path):
```

Comment:
I think it might be handy to pass the threshold to this method explicitly and set it to 0.5 by default, so clients can customise the behaviour if needed